internal/pkg/plugininstaller/reposcaffolding/option.go

```diff
@@ -51,7 +52,39 @@ func (opts *Options) CreateAndRenderLocalRepo(workpath string) error {
 
 // PushToRemoteGitLab push local repo to remote gitlab repo
 func (opts *Options) PushToRemoteGitlab(repoPath string) error {
-	// TODO: add gitlab push func
+	dstRepo := &opts.DestinationRepo
```

Comment:
I found a problem with `Options`, but can only comment on this line.

`required` does not make sense for non-pointer structs. Remove `required` or change to `*STRUCTNAME`